### PR TITLE
Total score in native cypher

### PR DESCRIPTION
**Pull request by vanderGoes**
@brense Ik heb ff de shortcut gepakt door het sorteren op totale aanbevelingsscore in native cypher te doen. Alleen weet ik nog niet of het foutloos werkt. Hoe kan ik het testen?
Voor JCypher moeten we waarschijnlijk wachten op de volgende release.

### CODE REVIEW

File: src/main/scala/com/partup/EventsApiService.scala [hunk 1]
```diff
@@ -48,13 +48,22 @@ class EventsApiActor(eventRouter: ActorRef, eventLogger: ActorRef, neo4j: ActorR
     val decodedUserId: String = decrypt(userId)
     val user = new JcNode("User")
     val team = new JcNode("Team")
+    val activeIn = new JcRelation("ACTIVE_IN")
```
Comment (wouterd) [inline src/main/scala/com/partup/EventsApiService.scala]: I think this should be `new JcRelation("activeIn")` looking at how you reference it in the native cypher.

**Comment by wouterd** [comment]
Is it possible to calculate this "total score" of the relation in the recommendation engine?
That way, you can just order on it in the query, instead of scoring them in the query.

**Comment by vanderGoes** [comment]
Nope, I asked GraphAware for this. But that is not yet an option. Thus I
chose the easy option. Expect it to be replaced later.

File: src/main/scala/com/partup/EventsApiService.scala (hunk 1 above)
Comment (vanderGoes) [inline src/main/scala/com/partup/EventsApiService.scala]: Damn yes, this is sloppy :(. I'm gonna replace the reference to stay consistent with the style.

**Comment by vanderGoes** [comment]
Good idea, lousy implementation. But it works now.

**Comment by vanderGoes** [comment]
@brense I've updated my branch with ft-recommended-partups but now I can't run the API anymore due to a failed test in EventsApiServiceSpec. Do you have the same problem?

**Comment by brense** [comment]
@vanderGoes Did you manage to run it yet? `EventsApiServiceSpec` contains the tests. We had some trouble getting data into the in-memory neo4j for the testing, so this is why the tests are failing.

**Comment by vanderGoes** [comment]
Wouter Fixed the tests somehow with his commit. Before (10f31b4) and after (49df2c0) everything was running smooth. I didn't look into the details

**Pull request merged**